hpx/util/memory_chunk_pool_allocator.hpp

```diff
@@ -106,6 +107,7 @@ namespace hpx { namespace util { namespace detail
         }
 
         Pool * memory_pool_;
+        bool   disable_deallocate;
```

Comment:
I believe this is left over from some earlier experimentation - the verbs PP passes pinned memory blocks to the parcel decode functions where they are freed, but there was a need to disable the deallocation so that the blocks could be reused. I suspect this is now obsolete and I will investigate furtuer and address this.